spec/lib/spec-helpers.ts

```diff
@@ -21,8 +21,6 @@ const addOnly = <T>(fn: Function): T => {
 export const ifit = (condition: boolean) => (condition ? it : addOnly<TestFunction>(it.skip));
 export const ifdescribe = (condition: boolean) => (condition ? describe : addOnly<SuiteFunction>(describe.skip));
 
-export const delay = (time: number = 0) => new Promise(resolve => setTimeout(resolve, time));
```

Comment:
```diff
diff --git a/spec/lib/spec-helpers.ts b/spec/lib/spec-helpers.ts
index 485f960f58..6c4d1a774a 100644
--- a/spec/lib/spec-helpers.ts
+++ b/spec/lib/spec-helpers.ts
@@ -181,6 +181,7 @@ export async function itremote (name: string, fn: Function, args?:
 any[]) {
     const { ok, message } = await w.webContents.executeJavaScript(`(async () => {
       try {
         const chai_1 = require('chai')
+        const promises_1 = require('timers/promises')
         chai_1.use(require('chai-as-promised'))
         chai_1.use(require('dirty-chai'))
         await (${fn})(...${JSON.stringify(args ?? [])})
```

fixes the webview spec failure

Reply:
wanna make that a suggestion on the right line in this PR? :)

Reply:
I can't - you can't make suggestions on lines of a given PR that aren't changed

Reply:
oh d'oh 🤦🏻‍♂️ thanks!

Reply:
i would have if i could have 😅